packages/ember-data/tests/unit/model-test.js

```diff
@@ -298,6 +297,68 @@ test("changedAttributes() return correct values", function() {
   equal(Object.keys(mascot.changedAttributes()).length, 0, 'after rollback attributes there are no changes');
 });
 
+test("changedAttributes() works while the record is being saved", function() {
+  var cat;
```

Comment:
Since the assert is in an adapter method could you add `expect(1)` to the start of this test.